pkg/lib/cockpit-components-password.jsx

```diff
@@ -120,19 +122,19 @@ export const PasswordFormFields = ({
                         <Button
                             variant="control"
                             onClick={() => setPasswordHidden(!passwordHidden)}
-                            aria-label={passwordHidden ? _("Show password") : _("Hide password")}>
+                            aria-label={ passwordHidden ? _("Show password") : _("Hide password")}>
```

Comment:
This is inconsistent spacing, and it also doesn't belong into the "lib: Change width of password progress" commit. This bit should be reverted.